components/mediation/inbound-endpoints/org.wso2.micro.integrator.inbound.endpoint/src/main/java/org/wso2/carbon/inbound/endpoint/protocol/rabbitmq/RabbitMQUtils.java

```diff
@@ -263,6 +263,8 @@ public static String buildMessage(AMQP.BasicProperties properties, byte[] body,
         // set content-type to the message context
         String contentType = properties.getContentType();
         if (contentType == null) {
+            log.warn("Unable to determine content type for message " + msgContext.getMessageID()
```

Comment:
Is content type a mandatory prop in rabbit mq ? If not this should be an info log ?